cmd/cephcsi.go

```diff
@@ -38,20 +38,20 @@ const (
 
 var (
 	// common flags
-	vtype           = flag.String("type", "", "driver type [rbd|cephfs]")
-	endpoint        = flag.String("endpoint", "unix://tmp/csi.sock", "CSI endpoint")
-	driverName      = flag.String("drivername", "", "name of the driver")
-	nodeID          = flag.String("nodeid", "", "node id")
-	metadataStorage = flag.String("metadatastorage", "", "metadata persistence method [node|k8s_configmap]")
+	vtype      = flag.String("type", "", "driver type [rbd|cephfs]")
+	endpoint   = flag.String("endpoint", "unix://tmp/csi.sock", "CSI endpoint")
+	driverName = flag.String("drivername", "", "name of the driver")
+	nodeID     = flag.String("nodeid", "", "node id")
 
 	// rbd related flags
 	containerized = flag.Bool("containerized", true, "whether run as containerized")
-	configRoot    = flag.String("configroot", "/etc/csi-config", "directory in which CSI specific Ceph"+
-		" cluster configurations are present, OR the value \"k8s_objects\" if present as kubernetes secrets")
+	instanceID    = flag.String("instanceid", "", "Unique ID distinguishing this instance of Ceph CSI among other"+
+		" instances, when sharing Ceph clusters across CSI instances for provisioning")
```

Comment:
@ShyamsundarR  Is it possible or well defined solution which allows same Ceph Cluster for multple provisioners?